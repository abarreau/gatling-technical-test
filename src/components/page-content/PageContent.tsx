import { FC, ReactNode } from 'react';
import styles from './PageContent.module.css';

/**
 * A page content is meant to be the root of your page, just after the PageHeader
 */
export const PageContent: FC<{ children?: ReactNode }> = ({ children }) => {
    return (<div className={styles.mainContent}>{ children }</div>);
};