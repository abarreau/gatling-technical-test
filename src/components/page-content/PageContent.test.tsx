import renderer from 'react-test-renderer';
import { PageContent } from './PageContent';

describe('PageContent', () => {
    it('should display component', () => {
        const component = renderer.create(
            <PageContent>My content</PageContent>
        );

        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
