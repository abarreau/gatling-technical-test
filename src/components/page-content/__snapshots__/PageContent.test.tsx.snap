// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageContent should display component 1`] = `
<div
  className="mainContent"
>
  My content
</div>
`;
